Sources/SharedServices/DynamoUserRepository.swift

```diff
@@ -84,39 +84,19 @@ struct DynamoUserRepository {
     }
     
     // MARK: - Link users
-    func linkGitHub(with discordId: String, _ githubId: String) async throws -> String {
-        // TODO: Implement
-        // Check if the users github already exists
-        
-        // If it exists, merge the 2 accounts
-        
-        // If the the given discordId already has a github account linked, overwrite the githubId
-        
-        // Delete the account that's not up-to-date
-        abort()
-    }
-    
-    func linkDiscord(with githubId: String, _ discordId: String) async throws -> String {
-        // TODO: Implement
-        // Check if the users discord already exists
-        
-        // If it exists, merge the 2 accounts
-        
-        // If the the given discordId already has a github account linked, overwrite the githubId
+
+    func linkGithub(with discordId: String, _ githubId: String) async throws -> String {
+        // TODO: Wip
+        let query = DynamoDB.QueryInput(
+            expressionAttributeValues: [":v1": .s("DISCORD-\(discordId)")],
+            indexName: discordIndex,
+            keyConditionExpression: "data1 = :v1",
+            limit: 1,
+            tableName: self.tableName
+        )
```

Comment:
I know you haven't yet gotten to this, but you just need to add their github id and then probably call `updateUser()`